appland-navie/index.html

```diff
@@ -1,58 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>AppMap AI: Explain</title>
-        <link rel="stylesheet" href="./dist/main.css" />
-        <style>
-            body {
-                font-family: system-ui, sans-serif;
-                height: 100%;
-                margin: 0;
-                overflow-y: hidden;
-            }
-
-            /* 
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>AppMap AI: Explain</title>
+    <link rel="stylesheet" href="./dist/main.css" />
+    <style>
+        body {
+            font-family: system-ui, sans-serif;
+            font-size: 80%;
```

Comment:
What's the reason for this change?

Reply:
No, this wasn't related to your review. Dan noticed that the text is much bigger than what's rendered in VS Code. This makes them much more similar in size.

Do you see a problem with this change?

Reply:
With this change, on my setup, Navie looked fine to me.
The IDE does have its own scaling factor for UI, but AFAIK it's not applied to the webviews. It's messy, unfortunately. 
There's also `zoom: ...` as a possible alternative  https://developer.mozilla.org/en-US/docs/Web/CSS/zoom.
I'm fine to merge this as it is. I'll keep an eye on the webview scaling on different setups.

Reply:
I considered `zoom`. It resizes everything in the viewport, rather than just the text, and I don't think we want to do that. Diagrams look ok to me.